website/docs/r/vapp_network.html.markdown

```diff
@@ -49,15 +48,20 @@ The following arguments are supported:
   connected as sysadmin working across different organisations.
 * `vdc` - (Optional; *v2.0+*) The name of VDC to use, optional if defined at provider level.
 * `name` - (Required) A unique name for the network.
+* `description` - (Optional; *v2.7+*) Description of vApp network
 * `vapp_name` - (Required) The vApp this VM should belong to.
 * `netmask` - (Optional) The netmask for the new network. Default is `255.255.255.0`.
 * `gateway` (Optional) The gateway for this network.
-* `dns1` - (Optional) First DNS server to use. Default is `8.8.8.8`.
-* `dns2` - (Optional) Second DNS server to use. Default is `8.8.4.4`.
+* `dns1` - (Optional) First DNS server to use.
+* `dns2` - (Optional) Second DNS server to use.
 * `dns_suffix` - (Optional) A FQDN for the virtual machines on this network.
 * `guest_vlan_allowed` (Optional) True if Network allows guest VLAN tagging. This value supported from vCD version 9.0
 * `static_ip_pool` - (Optional) A range of IPs permitted to be used as static IPs for virtual machines; see [IP Pools](#ip-pools) below for details.
 * `dhcp_pool` - (Optional) A range of IPs to issue to virtual machines that don't have a static IP; see [IP Pools](#ip-pools) below for details.
+* `org_network_name` - (Optional; *v2.7+*) An Org network name to which vApp network is connected to. If not configured, then isolated network created.
+* `firewall_enabled` - (Optional; *v2.7+*) Firewall service enabled or disabled. Default is true.
+* `nat_enabled` - (Optional; *v2.7+*) NAT service enabled or disabled. Default is true.
```

Comment:
Probably worth noting that `nat_enabled` won't work if `firewall_enabled=false`

Reply:
added